docs/src/index.md

```diff
@@ -107,9 +107,9 @@ projtoml = joinpath("..", "..", "Project.toml")
 version = TOML.parse(read(projtoml, String))["version"]
 name = TOML.parse(read(projtoml, String))["name"]
 link_manifest = "https://github.com/SciML/" * name * ".jl/tree/gh-pages/v" * version *
-       "/assets/Manifest.toml"
+                "/assets/Manifest.toml"
```

Comment:
This is what I would have thought/wanted the function argument above to get aligned as... But if the above is desirable then shouldn't this get indented just one tab spacing too?